crates/api_models/src/connector_enums.rs

```diff
@@ -123,6 +123,7 @@ pub enum Connector {
     //Thunes,
     Trustpay,
     Tsys,
+    // Unifiedauthenticationservice,
```

Comment:
Can we have this as `UnifiedAuthenticationService` so that the `snake_case` value used would be `unified_authentication_service`?

Reply:
Yes, @sumanmaji4  can you make this change!